tests/core/resource/test_views.py

```diff
@@ -485,11 +485,16 @@ def test_invalid_content_type_header_on_record_returns_415(self):
                                    status=415)
         self.assertEqual(resp.json['code'], 415)
         messages = (
-            "Content-Type header should be one of " +
-            "['application/merge-patch+json', 'application/json']",
-            "Content-Type header should be one of " +
-            "['application/json', 'application/merge-patch+json']")
-        self.assertIn(resp.json['message'], messages)
+            "Content-Type header should be one of [",
+            "'application/json-patch+json'",
+            ", ",
+            "'application/json'",
+            ", ",
+            "'application/merge-patch+json'",
+            "]")
+        for message in messages:
+            self.assertIn(message, resp.json['message'])
+        self.assertEquals(len("".join(messages)), len(resp.json['message']))
```

Comment:
There is something with this test, I can't suggest easily a better way, but it can be improved IMO :)